docs/quick-start/connecting-redis/page.md

```diff
@@ -28,7 +28,8 @@ Following configuration keys are required for Redis connectivity:
 * `REDIS_HOST`: It specifies the hostname or IP address of your Redis server.
 * `REDIS_PORT`: It specifies the port number on which your Redis server is listening. The default Redis port is 6379.
 
-```bash
+```dotenv
+# configs/.local.env
```

Comment:
In the doku it is written on a position that the tool looks for this '.local.env' 

On my test it also works.

Reply:
Yes, while running tests on local, `.local.env` would work.. But since the doku is not for any specific env, so suggested to have the same here as well.

Reply:
done with [ffef36a](https://github.com/gofr-dev/gofr/pull/573/commits/ffef36a47349f4a5136006f4369c256e9a7be5fa)